models/order.model.js

```diff
@@ -28,11 +28,11 @@ const orderSchema = mongoose.Schema({
     alternate_phone_number: {
         type: Number
     },
-    cart_id: {
+    product_ids: [{
         type: mongoose.Schema.Types.ObjectId,
-        ref: 'Cart',
+        ref: 'product',
```

Comment:
use the Capital in stead of lower case and update where necessary
ref: Product instead of ref: product
and mongoose('Product' instead of mongoose('product'

Reply:
done